File: Beecrowd/Beginner/1000 - 1099/1041 - Coordinates of a Point/Python/1041_CoordinatesofaPoint.py
pointX, pointY = map(float, input().split())

if pointX == 0 and pointY == 0:
    print("Origem")
elif pointX == 0:
    print("Eixo Y")
elif pointY == 0:
    print("Eixo X")
elif pointX > 0 and pointY > 0:
    print("Q1")
elif pointX < 0 and pointY > 0:
    print("Q2")
elif pointX < 0 and pointY < 0:
    print("Q3")
else:
    print("Q4")

File: Beecrowd/1 - Beginner/1000 - 1099/1041 - Coordinates of a Point/1041_CoordinatesofaPoint.c
#include <stdio.h>

int main() {
    double pointX, pointY;
    scanf("%lf %lf", &pointX, &pointY);

    if (pointX == 0 && pointY == 0) printf("Origem\n");
    else if (pointX == 0) printf("Eixo Y\n");
    else if (pointY == 0) printf("Eixo X\n");
    else if (pointX > 0 && pointY > 0) printf("Q1\n");
    else if (pointX < 0 && pointY > 0) printf("Q2\n");
    else if (pointX < 0 && pointY < 0) printf("Q3\n");
    else printf("Q4\n");
    
    return 0;
}
